### PR TITLE
Feature/kndp 30 helm charts packaging action

### PR DESCRIPTION
**Pull request by Timex1854**

### CODE REVIEW

File: .github/workflows/chart-releaser.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          charts_dir: packages/kndp/charts
+
+        env:
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          CR_GIT_REPO: helm-charts
+          CR_OWNER: kndp
+          CR_GIT_BASE_URL: "https://api.github.com/"
+          CR_GIT_UPLOAD_URL: "https://uploads.github.com/"
+          CR_SKIP_EXISTING: true
+          CR_PAGES_BRANCH: "release/0.1"
```
Comment (web7evgheni) [inline .github/workflows/chart-releaser.yaml:37]: GitHub pages branch is gh-pages

File: .github/workflows/chart-releaser.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        uses: azure/setup-helm@v3
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.4.1
+        with:
+          charts_dir: packages/kndp/charts
+
+        env:
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          CR_GIT_REPO: helm-charts
+          CR_OWNER: kndp
```
Comment (web7evgheni) [inline .github/workflows/chart-releaser.yaml:33]: Owner is web-seven

**Review by web7evgheni — CHANGES_REQUESTED** [review]

File: .github/workflows/chart-releaser.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+name: Release Charts
+on:
+  pull_request:
+    types:
```
[the remark this answers is not in the document]
Comment (Timex1854) [reply .github/workflows/chart-releaser.yaml:4]: Executing the workflow at any push in any branch?

[the remark this answers is not in the document]
Comment (web7evgheni) [reply .github/workflows/chart-releaser.yaml:4]: Yes

**Review by web7evgheni — APPROVED** [review]

**Pull request merged**